lib/taiko.js

```diff
@@ -89,6 +90,7 @@ module.exports.openBrowser = async (
   if ((options.host && options.port) || options.target) {
```

Comment:
@ankur-lt `useHostName` will be set only when host, port or target is set. Is this expected or only `useHostName` with openBrowser is good.

Reply:
Yes, this is expected. It will be used only when host, port or target is specified.